.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,20 @@
-## Why are you doing this?
+<!-- all sections optional, delete any you don't need -->
+## What are you doing in this PR?
 
 <!--
-Remember, PRs are documentation for future contributors.
-
-If this PR is a fix, please include a link to the original PR that introduced
-the breakage for reference.
+This pr adds a widget to the doogle so that we can buy a sub from any page in one click
+For detailed changes see the inline self-review comments.
 -->
 
-[**Trello Card**](https://trello.com)
-
-## Changes
+## Why are you doing this?
 
-* Change 1
-* Change 2
+<!--
+Remember, PRs are documentation for future contributors.
+-->
 
-## Did you enjoy your PR experience?
+## Useful links (trello, google docs, previous PRs, etc)
 
- - [ ] [PR experience rated](https://forms.gle/N6FsTGG8JQFGV4Ha9)
+<!-- trello -->
```

Comment:
the existing trello link is finickity and it's easier to just paste the link as it comes

Reply:
you like it to be specifically a [....](...) style link, or you just want it to be prominent and specific?

It was the [...](...) that always caused me hassle - but I just thought, I always carefully paste the link inside the (....) but there's nothing stopping me personally pasting over it completely every time!